plugin.video.mediathekview/resources/lib/mvupdate.py

```diff
@@ -13,405 +13,45 @@
-        """
-        pass
-
-    @staticmethod
-    def close_progress_dialog():
-        """
-        Closes a progress dialog
-        """
-        pass
-
-    @staticmethod
-    def show_database_error(err):
-        """ Displays UI for a database error """
-        pass
-
-    @staticmethod
-    def show_download_error(name, err):
-        """ Displays UI for a download error """
-        pass
-
-    @staticmethod
-    def show_missing_extractor_error():
-        """ Disaplys UI for a missing extractor error """
-        pass
-
-    @staticmethod
-    def show_limit_results(maxresults):
-        """ Display UI for search result limited by configuration """
-        pass
-
-    @staticmethod
-    def show_outdated_unknown():
-        """ Display UI for never updated database """
-        pass
-
-    @staticmethod
-    def show_outdated_known(status):
-        """ Display UI for an outdated database """
-        pass
-
-    @staticmethod
-    def show_download_progress():
-        """ Display UI for a download in progress """
-        pass
-
-    @staticmethod
-    def update_download_progress(percent, message=None):
-        """ Update UI odometer for a download in progress """
-        pass
-
-    @staticmethod
-    def hook_download_progress(blockcount, blocksize, totalsize):
-        """ UI Report hook for functions like `url_retrieve` """
-        pass
-
-    @staticmethod
-    def close_download_progress():
-        """ Hides the UI for a download in progress """
-        pass
-
-    @staticmethod
-    def show_update_progress():
-        """ Display UI for a database update in progress """
-        pass
-
-    @staticmethod
-    def update_update_progress(percent, count, channels, shows, movies):
-        """ Update UI odometer for a database update in progress """
-        pass
-
-    @staticmethod
-    def close_update_progress():
-        """ Hides the UI for a database update in progress """
-        pass
-
-    @staticmethod
-    def show_updating_scheme():
-        """ SHow UI that the database schema is about to be updated """
-        pass
-
-    @staticmethod
-    def show_update_scheme_progress():
-        """ Display UI for a database schema update in progress """
-        pass
-
-    @staticmethod
-    def update_update_scheme_progress(percent):
-        """ Update UI odometer for a database schema update in progress """
-        pass
-
-    @staticmethod
-    def close_update_scheme_progress():
-        """ Hides the UI for a database schema update in progress """
-        pass
-
-
-class MediathekViewMonitor(object):
-    """ Standalone implementation of the monitor class """
-    @staticmethod
-    def abort_requested():
-        """
-        Returns `True`if either this instance is not the registered
-        instance or Kodi is shutting down
-        """
-        return False
-
-
-class UpdateApp(AppLogger):
+class UpdateApp():
     """ The standalone updater application class """
```

Comment:
Sorry You'll have to drop this. Add-ons are not allowed to parse the add-on.xml nor to control any update mechanisms aside from kodi